### ISSUE TITLE
Cannot clear memory cache

### ISSUE THREAD

**Issue by edwardhuerta** [VictorAlbertos/RxCache#54]
Hi,

So I am using this framework in an app that requires me to flush the cached memory data when a user logs out. 

Is there an easy way of doing this?

The only way I have found so far is that I use the annotation Actionable and explicitly evictAll. But this poses a problem since the Actionable annotation can only be applied to results of List<>, leaving my other non-List<> data unable to be evicted.

**Comment by VictorAlbertos** [comment]
Hi @edwardhuerta 

This topic has been already discussed [here](https://github.com/VictorAlbertos/RxCache/issues/10). 

But probably I'll end up adding some support to RxCache for clearing the cached data. Eventually.

**Comment by VictorAlbertos** [comment]
Hi @edwardhuerta. I've added support for this feature.

It's available from version 1.6.1. 

To use it:

```java
RxCache rxCache = new RxCache.Builder()
                            .persistence(getFilesDir(), new GsonSpeaker());

Providers providers = rxCache.using(Providers.class);
rxCache.evictAll().subscribe();
```
